specification/Metadata/README.md

```diff
@@ -21,283 +21,227 @@ Draft
+> **Example:** This example defines a `car` class with three array-like properties. The `passengers` property is a variable-length array, because `componentCount` is undefined.
+>
+> | property         | description                |  type   | componentType | componentCount |
+> |:-----------------|:---------------------------|:-------:|:-------------:|---------------:|
+> | forwardDirection | "Forward direction vector" | "VEC3"  |   "FLOAT64"   |              3 |
+> | passengers       | "Passenger names"          | "ARRAY" |   "STRING"    |                |
+> | modelMatrix      | "4x4 model matrix"         | "MAT4"  |   "FLOAT32"   |             16 |
 
-In the example below each tree can be classified as one of four tree species.
+#### Component Type
 
-```jsonc
-{
-  "schema": {
-    "enums": {
-      "species": {
-        "valueType": "INT8",
-        "values": [
-          {
-            "name": "Oak",
-            "value": 0
-          },
-          {
-            "name": "Pine",
-            "value": 1
-          },
-          {
-            "name": "Maple",
-            "value": 2
-          },
-          {
-            "name": "Unknown",
-            "value": -1
-          }
-        ]
-      }
-    },
-    "classes": {
-      "tree": {
-        "properties": {
-          "height": {
-            "type": "FLOAT32"
-          },
-          "species": {
-            "type": "ENUM",
-            "enumType": "species"
-          },
-          "leafColor": {
-            "type": "STRING",
-            "optional": true,
-            "default": "green"
-          }
-        }
-      }
-    }
-  }
-}
-```
+Properties may be comprised of one component (`SINGLE`) or many components (`ARRAY`, `VECN`, `MATN`), depending on the property `type`. Each component is an instance of the property's component type (`componentType`), with the following component types supported:
 
-### Normalized Properties
+| name    | componentType                                                             |
+|---------|---------------------------------------------------------------------------|
+| INT8    | Signed integer in the range `[-128, 127]`                                 |
+| UINT8   | Unsigned integer in the range `[0, 255]`                                  |
+| INT16   | Signed integer in the range `[-32768, 32767]`                             |
+| UINT16  | Unsigned integer in the range `[0, 65535]`                                |
+| INT32   | Signed integer in the range `[-2147483648, 2147483647]`                   |
+| UINT32  | Unsigned integer in the range `[0, 4294967295]`                           |
+| INT64   | Signed integer in the range `[-9223372036854775808, 9223372036854775807]` |
+| UINT64  | Unsigned integer in the range `[0, 18446744073709551615]`                 |
+| FLOAT32 | A number that can be represented as 32-bit IEEE floating point            |
+| FLOAT64 | A number that can be represented as 64-bit IEEE floating point            |
```

Comment:
we probably should be clearer that `NaN` and `Infinity` are excluded

Reply:
Added the restriction just below the table. 👍